recipe/meta.yaml

```diff
@@ -12,35 +12,39 @@ source:
   sha256: {{ sha256sum }}
 
 build:
-  noarch: python
-  number: 0
+  number: 1
   script: {{ PYTHON }} -m pip install . --no-deps -vv
 
 requirements:
   host:
     - pip
     - python
     - setuptools
+    - wheel
+    - yajl
 
   run:
     - python
+    - yajl
 
 test:
   imports:
     - ijson
     - ijson.backends
   commands:
     - pip check
+    - {{ PYTHON }} -c "import ijson; assert ijson.backend == 'yajl2_c'"
```

Comment:
I think there should be `assert ijson.backends == 'yajl2_c'"`
But do we need to enable `yajl` as an extension?

Reply:
The test worked, it finds the backend automatically as long as yajl is installed and the C extension was built. Trying calling python in a different way.